command/rootchain/fund/params.go

```diff
@@ -9,10 +9,10 @@ import (
 )
 
 const (
-	addressesFlag     = "addresses"
-	amountsFlag       = "amounts"
-	jsonRPCFlag       = "json-rpc"
-	mintRootTokenFlag = "mint"
+	addressesFlag      = "addresses"
+	amountsFlag        = "amounts"
+	jsonRPCFlag        = "json-rpc"
+	mintStakeTokenFlag = "mint"
```

Comment:
Should this be just `mint` flag?

Reply:
I've renamed it to paint a better a picture what is minted.